addons/payment/wizards/payment_acquirer_onboarding_wizard.py

```diff
@@ -158,3 +141,61 @@ def action_onboarding_other_payment_acquirer(self):
         self._set_payment_acquirer_onboarding_step_done()
         action = self.env["ir.actions.actions"]._for_xml_id("payment.action_payment_acquirer")
         return action
+
+    def _init_paypal(self, env):
+        env.ref('payment.payment_acquirer_paypal').write({
+            'paypal_email_account': self.paypal_email_account,
+            'paypal_seller_account': self.paypal_seller_account,
+            'paypal_pdt_token': self.paypal_pdt_token,
+            'state': 'enabled',
+        })
+
+    def _init_stripe(self, env):
+        if self.stripe_publishable_key and self.stripe_secret_key:
+            env.ref('payment.payment_acquirer_stripe').write({
+                'stripe_secret_key': self.stripe_secret_key,
+                'stripe_publishable_key': self.stripe_publishable_key,
+                'state': 'enabled',
+                'journal_id': self.env['account.journal'].search([
+                    ('type', '=', 'bank'), ('company_id', '=', env.company.id)
+                ], limit=1).id,
+            })
+        else:
+            env.ref('payment.payment_acquirer_stripe').write({
```

Comment:
Yes, cause if we enable it, we'll have a ValidationError since neither Stripe Secret/Publishable Key (`stripe_secret_key`, `stripe_publishable_key`) are filled, neither Stripe Account Id (`stripe_account_id`).

The acquirer will be enabled when the user comes back from a completed Stripe Onboarding.